src/pages/ProjectSettings.tsx

```diff
@@ -113,7 +114,12 @@ export const ProjectSettings: React.FC = () => {
   return (
     <ProjectSettingsProvider>
       <SideNav aria-label="Project Settings">
-        <SideNavGroup header="Project" />
+        <DropdownContainer>
```

Comment:
Can we not show "Favorites" if no favorites are selected?
![image](https://user-images.githubusercontent.com/26798134/156177444-fc48f479-35a5-4561-ab00-94e1dfa46119.png)

Since unattached projects don't have a header, it makes them all look like favorites.